gammapy/spectrum/results.py

```diff
@@ -0,0 +1,467 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+from astropy.table import Table, Column, QTable
+from astropy.units import Unit
+
+from gammapy.extern.bunch import Bunch
+from gammapy.utils.energy import EnergyBounds
+from gammapy.utils.scripts import read_yaml
+
+
+class SpectrumFitResult(object):
+    """Class representing the result of a spectral fit
+
+    For a complete documentation see :ref:`gadf:spectrum-fit-result`
+
+    Parameters
+    ----------
+    spectral_model : str
+        Spectral model, for allowed names see :ref:`gadf:source-models`
+    parameters : dict
+        Fitted parameters, for allowed names see :ref:`gadf:source-models`
+    parameter_errors : dict
+        Parameter errors, for allowed names see :ref:`gadf:source-models`
+    energy_range : `~gammapy.utils.energy.EnergyBounds
+        Energy range of the spectral fit
+    fluxes : dict, optional
+        Flux for the fitted model at a given energy
+    flux_errors : dict, optional
+        Error on the flux for the fitted model at a given energy
+    flux_points : `~astropy.table.Table`, optional
+        Flux points
+    """
+    def __init__(self, spectral_model, parameters, parameter_errors,
+                 energy_range=None, fluxes=None, flux_errors=None,
+                 flux_points=None):
+        self.spectral_model = spectral_model
+        self.parameters = Bunch(parameters)
+        self.parameter_errors = Bunch(parameter_errors)
+        self.energy_range = EnergyBounds(energy_range).to('TeV')
+        self.fluxes = fluxes
+        self.flux_errors = flux_errors
+        self.flux_points = Table(flux_points)
+
+    @classmethod
+    def from_fitspectrum_json(cls, filename, model=0):
+        """Read FitSpectrum output file
+
+        Several models can be stored in the JSON file, the appropriate model
+        to read can be chosen with the ``model`` parameter.
+        TODO: Should this be a function?
```

Comment:
Ah, OK.
Maybe change the TODO from "Should this be a function" to "Remove method when HESS FitSpectrum exporter is updated" then?
Some iteration on the serialisation format is needed for the coming months ... in the meantime it's fully OK to have such converter and convenience functions / methods in Gammapy.